src/site/pages/guidelines.md

```diff
@@ -64,77 +64,76 @@ PatternFly follows a two-layer theming system where **global variables** always
 The main reason to have global variables is to maintain consistency. They adhere to the following rules:
 
 - They are prefixed with the word `global` and follow the formula `--pf-global--concept--PropertyCamelCase--modifier--state`.
-  - a `concept` is something like a `spacer` or `main-title`;
+  - a `concept` is something like a `spacer` or `main-title`.
   - a `PropertyCamelCase` is something like `BackgroundColor` or `FontSize`.
-  - a `modifier` is something like  `sm`, or `lg`;
-  - and a `state` is something like  `hover`, or `expanded`;
+  - a `modifier` is something like  `sm`, or `lg`.
+  - and a `state` is something like  `hover`, or `expanded`.
 - They are concepts, never tied to an element or component. This is incorrect: `--pf-global--h1--FontSize`, this is correct: `--pf-global--FontSize--3xl`.
 
 For example a global variable setup would look like:
 
 ```scss
-
-  /* --pf-global--concept--size */
+  // --pf-global--concept--size
   --pf-global--spacer--lg: .5rem;
   --pf-global--spacer--xl: 1rem;
   --pf-global--spacer--2xl: 2rem;
 
-  /* --pf-global--PropertyCamelCase--modifier */
+  // --pf-global--PropertyCamelCase--modifier
   --pf-global--FontSize--3xl: 2rem;
   --pf-global--FontSize--2xl: 1.8rem;
   --pf-global--FontSize--lg: 1rem;
 
-  /* --pf-global--PropertyCamelCase--state */
-  --pf-global--BackgroundColor--hover: #ccc;
+  // --pf-global--PropertyCamelCase--state
+  --pf-global--link--TextDecoration--hover: #ccc;
 
-  /* --pf-global--PropertyCamelCase--modifier */
+  // --pf-global--PropertyCamelCase--modifier
   --pf-global--Color--100: #000;
 
-  /* --pf-global--concept--modifier */
-  --pf-global--primary-color--100: blue;
+  // --pf-global--concept--modifier
+  --pf-global--primary-color--100: #00f;
 ```
 
 ### Component variables
 
 The second layer is scoped to themeable component custom properties. This setup allows for consistency across components, generates a common language between designers and developers, and gives granular control to authors. The rules are as follows:
 
-- They follow this general formula `--pf-c-block__element--modifier--state--breakpoint--pseudo-element--PropertyCamelCase`.
+- They follow this general formula `--pf-c-block[__element][--modifier][--state][--breakpoint][--pseudo-element]--PropertyCamelCase`.
   - `pf-c-block` refers to the block, usually the component or layout name (i.e., `pf-c-alert`).
   - `__element` refers to the element inside of the block (i.e., `__title`).
```

Comment:
`__element` is the only piece of syntax that's illustrated with the decorating dashes/underscores. To be consistent we could remove them from this one so it just reads `element` like the others, or we could add to the other examples their corresponding underscores/dashes like `--pf-c-block`, `--modifier`, etc. I think the latter is more literal and leaves little to be misunderstood, but using either approach across the board would be better regardless of which one we use.

Love the examples for each one btw, really helps.